kubevirtci

```diff
@@ -96,9 +96,9 @@ function kubevirtci::install() {
 }
 
 function kubevirtci::create_cluster() {
-	export NODE_VM_IMAGE_TEMPLATE=quay.io/kubevirtci/fedora-kubeadm:35
+	export NODE_VM_IMAGE_TEMPLATE=quay.io/isaacdorfman/kubevirt-kubernetes-node
 	export IMAGE_REPO=k8s.gcr.io
-	export CRI_PATH="/var/run/crio/crio.sock"
+	export CRI_PATH="/var/run/containerd/containerd.sock"
```

Comment:
Can you please elaborate, why was this changed?

Reply:
The new image uses containerd instead of cri-o.
And containerd listens on the "/var/run/containerd/containerd.sock" socket.

Reply:
👍